scripts/image-build.sh

```diff
@@ -0,0 +1,32 @@
+
+#!/bin/bash
+
+# This script builds the candid docker image and pushes it to the
```

Comment:
Java world 1, rest of us 0: https://github.com/GoogleContainerTools/jib :D jokes aside LGTM but would suggest buildkit for secrets & keys